recipe/yum_requirements.txt

```diff
@@ -0,0 +1,5 @@
+java-1.8.0-openjdk-devel
+unzip
+zip
+zlib-devel
```

Comment:
Yup, Bazel seems to ignore the conda installed zlib, so the yum zlib-devel is required.  There may be a method to adjust Bazel' search path but I had not been able to figure out how to do this.